src/x86/x86_generator.py

```diff
@@ -324,18 +324,26 @@ def run_on_test_case(self, test_case: TestCase) -> None:
                         assert len(mem_operands) == 1, f"Unexpected instruction format {instr.name}"
                         mem_operand: Operand = mem_operands[0]
                         registers = mem_operand.value
-
-                        offsets = ["RCX", "RDX"]
-                        masks = ["RAX", "RBX"]
-                        offset_reg = offsets[0]
-                        mask_reg = masks[0]
-                        for reg in src_operands:
+                        
+                        masksList = ["RAX", "RBX"]
```

Comment:
Code style issue: we use the style `lower_case_with_underscores` for variables

Reply:
See here for more details: https://peps.python.org/pep-0008/#function-and-variable-names